scripts/coredump/coredump_gdbserver.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import logging
+import os
+import socket
+import sys
+
+from parser.log_parser import CoredumpLogFile
+from parser.elf_parser import CoredumpElfFile
+
+import gdbstubs
+
+LOGGING_FORMAT = "[%(levelname)s][%(name)s] %(message)s"
+
+GDBSERVER_HOST = ""
+GDBSERVER_PORT = 1234
```

Comment:
very very minor, but the port value could be an argument, that is the default port and one may already being debugging a qemu instance, for example.

Reply:
agreed we should make this an argument, with a default of 1234

Reply:
Made this as an argument.